apps/web/src/utils/importer.ts

```diff
@@ -198,10 +161,20 @@ async function processNote(entry: ZipEntry, attachments: Record<string, any>) {
   }
 }
 
-async function fileToJson<T>(file: ZipEntry) {
-  const text = await file.text();
-  return JSON.parse(text) as T;
-}
+const colorMap: Record<string, string | undefined> = {
+  default: undefined,
+  teal: "#00897B",
+  red: "#D32F2F",
+  purple: "#7B1FA2",
+  blue: "#1976D2",
+  cerulean: "#03A9F4",
+  pink: "#C2185B",
+  brown: "#795548",
+  gray: "#9E9E9E",
+  green: "#388E3C",
+  orange: "#FFA000",
+  yellow: "#FFC107"
+};
```

Comment:
Let's move this to the top. It's kind of a convention:

1. Types
2. Constants
3. Functions